jxp/My-List-Util/t/sum.t

```diff
@@ -0,0 +1,18 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More tests => 1;
```

Comment:
Yes, but if I change it to 2 it states only 1 has run. If you change the expected value it fails, so I know it's checking those correctly.